contracts/gateway/AxelarAmplifierGateway.sol

```diff
@@ -92,54 +88,65 @@ contract AxelarAmplifierGateway is IAxelarAmplifierGateway {
     |* External Functions *|
     \**********************/
 
-    function execute(SignedCommandBatch calldata signedBatch) external {
-        CommandBatch calldata batch = signedBatch.batch;
-        bytes memory data = abi.encode(batch);
-
-        // This is the EVM convention for signing non-tx data to domain separate from regular EVM txs.
-        // This should be customized per chain.
-        bytes32 batchHash = ECDSA.toEthSignedMessageHash(keccak256(data));
-
-        bool isLatestSigners = authModule.validateProof(batchHash, signedBatch.proof);
+    function verifyMessages(SignedMessageBatch calldata signedBatch) public {
```

Comment:
I am still a lil bit confused by the `verifyMessages` method name as we don't actually verify any messages in this context.
How about:
`deliverMessages`
`relayMessages`
`approveMessages` +1
`approveBatch`
`passMessages` hence GMP?

Reply:
`verifyMessages` comes from Amplifier gateways. Not settled on it yet